govcd/vapp.go

```diff
@@ -145,6 +145,13 @@ func (vapp *VApp) AddVM(orgVdcNetworks []*types.OrgVDCNetwork, vappNetworkName s
 
 // AddNewVM adds VM from vApp template with custom NetworkConnectionSection
 func (vapp *VApp) AddNewVM(name string, vappTemplate VAppTemplate, network *types.NetworkConnectionSection, acceptAllEulas bool) (Task, error) {
+	return vapp.AddNewVMWithStorage(name, vappTemplate, network, nil, acceptAllEulas)
```

Comment:
Imho, I would like with `StorageProfile`. Atleast for me first thought was disk - not storage profile.